cargo-shuttle/src/lib.rs

```diff
@@ -392,9 +416,21 @@ impl Shuttle {
             true
         } else {
             let should_create = Confirm::with_theme(&theme)
-                .with_prompt("Do you want to create the project environment on Shuttle?")
+                .with_prompt(format!(
+                    r#"Claim the project name "{}" by starting a project container on Shuttle?"#,
+                    project_args
+                        .name
+                        .as_ref()
+                        .expect("to have a project name provided")
+                ))
                 .default(true)
                 .interact()?;
+            if !should_create {
+                println!(
+                    "Note: The project name will not be claimed until \
+                    you start the project with `cargo shuttle project start`."
+                )
+            }
```

Comment:
Will it be clear by this point that the name is globally unique?

Reply:
The prompt at name selection does tell you that. The note here is to clarify for those that do not claim the name. The word "claim" also helps.